setup.py

```diff
@@ -39,7 +39,7 @@ def get_packages():
 
 setup(
     name='lettuce',
-    version='0.2.22',
+    version='0.2.23',
```

Comment:
Might want to use something like `0.2.22b` or `0.2.22.1` − it's ugly anyway, but at least we don't risk conflicting with the upstream repo